app/models/user.js

```diff
@@ -1,46 +1,66 @@
-import attr from 'ember-data/attr';
-import Ember from 'ember';
-import { hasMany } from 'ember-data/relationships';
-import Addressable from './addressable';
+import attr from "ember-data/attr";
+import Ember from "ember";
+import { hasMany } from "ember-data/relationships";
+import Addressable from "./addressable";
 
 export default Addressable.extend({
-  firstName:   attr('string'),
-  lastName:    attr('string'),
-  mobile:      attr('string'),
-  createdAt:   attr('date'),
-  email:       attr('string'),
+  firstName: attr("string"),
+  lastName: attr("string"),
+  mobile: attr("string"),
+  createdAt: attr("date"),
+  email: attr("string"),
   organisations_users_ids: attr(),
-  lastConnected: attr('date'),
-  lastDisconnected: attr('date'),
+  lastConnected: attr("date"),
+  lastDisconnected: attr("date"),
   i18n: Ember.inject.service(),
 
   // permission:  belongsTo('permission', { async: false }),
   // image:       belongsTo('image', { async: false }),
 
-  userRoles: hasMany('userRoles', { async: false }),
-  roles: hasMany('roles', { async: false }),
+  userRoles: hasMany("userRoles", {
+    async: false
+  }),
+  roles: hasMany("roles", {
+    async: false
+  }),
 
-  fullName: Ember.computed('firstName', 'lastName', function(){
-    return (this.get('firstName') + " " + this.get('lastName'));
+  fullName: Ember.computed("firstName", "lastName", function() {
+    return this.get("firstName") + " " + this.get("lastName");
   }),
 
-  organisations: hasMany('organisation', {async: false}),
-  organisationsUsers: hasMany('organisationsUsers', {async: false}),
+  organisations: hasMany("organisation", {
+    async: false
+  }),
+  organisationsUsers: hasMany("organisationsUsers", {
+    async: false
+  }),
 
-  position: Ember.computed('organisationsUsers.[]', function() {
+  position: Ember.computed("organisationsUsers.[]", function() {
     const organisationsUsers = this.get("organisationsUsers");
-    return organisationsUsers.length ? 
-    organisationsUsers.get("firstObject.position") : '';
+    return organisationsUsers.length
+      ? organisationsUsers.get("firstObject.position")
+      : "";
+  }),
+
+  preferredContactNumber: Ember.computed("organisationsUsers.[]", function() {
+    let organisationsUser = this.get("organisationsUsers.firstObject");
```

Comment:
Instead of `firstObject`, is there any other way? Maybe filter by `currentUserId` ?

Reply:
nope.  It should not work that way as logged in user will be staff member and we want preferred_contact_number of charity user.

created_by will be always for whom we are creating order. thus we will take that users preferred_ccontact_number which is stored in organisations_user record.

Reply:
Okay makes sense. 👍